<launch>
    <arg name="configfile" default="" />

    <node name="ff_lins_node" pkg="ff_lins" type="ff_lins_ros" output="screen">
        <param name="imu_topic" value="/livox/imu" />
        <param name="lidar_topic" value="/livox/lidar" />

        <!-- MCD-Ouster64 -->
        <!-- <param name="imu_topic" value="/vn200/imu"/>
        <param name="lidar_topic" value="/os_cloud_node/points"/> -->

        <!-- Velodyne-->
        <!-- <param name="imu_topic" value="/imu0"/>
        <param name="lidar_topic" value="/velodyne_points"/> -->

        <param name="configfile" value="$(arg configfile)" />

        <param name="is_read_bag" value="fase" />
        <param name="bagfile" value="" />
    </node>

    <node pkg="tf2_ros" type="static_transform_publisher" name="world_to_map_broadcaster"
        args="0 0 0 -1 0 0 0 map world" />
    <node name="visualisation" pkg="rviz" type="rviz" output="log"
        args="-d $(find ff_lins)/config/visualization.rviz" />

</launch>